File: 1580-shuffle-the-array/1580-shuffle-the-array.cpp
class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
        int i = 0 ; 
        vector<int> ans ;
        while(i < n){
            int j = i + n ;
            ans.push_back(nums[i]);
            ans.push_back(nums[j]);
            i++;
        }
        return ans;
    }
};

File: 1580-shuffle-the-array/1580-shuffle-the-array.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int[] ans = new int[2*n] ;
        int ind = 0 ;
        for(int i = 0 ; i < 2*n ; i += 2){
            ans[i] = nums[ind] ;
            ans[i+1] = nums[ind + n] ;
            ind++ ;
        }
        return ans ;
    }
}